doc/sphinx/_static/notations.css

```diff
@@ -165,10 +166,41 @@
 /* Overrides */
 /*************/
 
-.rst-content table.docutils td, .rst-content table.docutils th {
-    padding: 8px; /* Reduce horizontal padding */
-    border-left: none;
-    border-right: none;
+
+.rst-content table.docutils.prodn {
+    border-style: none;
+}
+
+.rst-content table.docutils.prodn td,
+.rst-content table.docutils.prodn th {
+    border-style: none;
+    padding: 0px;
+    margin: 0.15em 0px;
+    vertical-align: baseline;
+    height: auto;
+    font-weight: bold;
+}
+
+.rst-content table.docutils.prodn tr:nth-child(2n-1) td {
+    background-color: #fcfcfc;
+}
+
+.rst-content table.prodn .notation > .repeat-wrapper {
+    margin-top: 0.28em;
+}
+
+.rst-content table.prodn .notation > .repeat-wrapper-with-sub {
+    margin-top: 0.28em;
+    margin-bottom: 0.28em;
```

Comment:
What do these constants correspond to?

Reply:
They are the top and bottom margins (in orange), intended to assure correct separation between adjacent lines in the table:

![image](https://user-images.githubusercontent.com/1253341/70193912-884f2000-16b5-11ea-8e80-a062d587ec2a.png)

I tried putting margins on the notation-sup and notation-sub, but that did the wrong thing, moving each closer to the midline.